src/main/java/com/ldtteam/structurize/generation/DataGeneratorConstants.java

```diff
@@ -45,9 +46,13 @@
      */
     public static JsonElement serialize(IJsonSerializable serializable)
     {
-        JsonElement jsonElement = serializable.serialize();
-        if (jsonElement.isJsonObject())
-            jsonElement.getAsJsonObject().addProperty("READ ME!!!", "DO NOT EDIT THIS FILE. DON'T TOUCH IT, TOUCH IT AND YOU WILL REGRET IT ~~ AshersLab (Use the Data Generators!");
-        return jsonElement;
+        return serialize(serializable.serialize());
+    }
+
+    public static JsonElement serialize(JsonElement serialized)
+    {
+        if (serialized.isJsonObject())
+            serialized.getAsJsonObject().addProperty("READ ME!!!", "DO NOT EDIT THIS FILE. DON'T TOUCH IT, TOUCH IT AND YOU WILL REGRET IT ~~ AshersLab (Use the Data Generators!");
```

Comment:
the missing closing bracket (that i failed to put in the first place) is bugging me here =P

Reply:
I think you mean "(that I failed to put on in the first place"

Reply:
damn you